Fix/issue wooship 1438 - Fix the Arizona Tax Calculation for the Order placed in Arizona (#2854)
Automattic/woocommerce-services: changelog.txt

*** WooCommerce Tax Changelog ***

= 3.0.4 - 2025-xx-xx =

> * Fix   - Arizona tax calculation for the store located in Arizona and ship the product within Arizona.

Reviewer: ```suggestion
* Fix   - Corrected tax calculation for orders shipped within Arizona from stores based in Arizona.
```
